src/main/resources/course.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The requirement was to create xml file for at least 5 classes from the hierarchy.I dont see five classes included in your xml file.

Reply:
No associated xsd file is included.